### PR TITLE
Update choose-domain-to-create-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note that the scenario 2 works only with the MX points to EOP and not to any 3rd party spam filter.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8944

File: microsoft-365/solutions/choose-domain-to-create-groups.md [hunk 1]
```diff
@@ -120,4 +122,4 @@ There are a few more things to know:
 
 [Create your collaboration governance plan](collaboration-governance-first.md) (article)
 
-[Create an Microsoft 365 group in the admin center](../admin/create-groups/create-groups.md) (article)
\ No newline at end of file
+[Create an Microsoft 365 group in the admin center](../admin/create-groups/create-groups.md) (article)
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/choose-domain-to-create-groups.md]:
```suggestion
[Create a Microsoft 365 group in the admin center](../admin/create-groups/create-groups.md) (article)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**